server/task/task-memory/src/main/scala/org/apache/james/task/eventsourcing/TaskAggregate.scala

```diff
@@ -42,14 +41,7 @@ class TaskAggregate private(val aggregateId: TaskAggregateId, private val histor
     .asScala
     .foldLeft(DecisionProjection.empty)((decision, event) => decision.update(event))
```

Comment:
in fact there's no such empty projection concept as you can't have an aggregate without a Created event in the first place.
We should drop it to simplify the code